src/app/modules/vault/modules/organization-vault/organization-vault.component.ts

```diff
@@ -127,6 +128,8 @@ export class VaultComponent implements OnInit, OnDestroy {
     this.ciphersComponent.showAddNew = vaultFilter.status !== "trash";
     this.activeFilter = vaultFilter;
     await this.ciphersComponent.reload(this.buildFilter(), vaultFilter.status === "trash");
+    this.vaultFilterComponent.searchPlaceholder =
+      this.vaultService.calculateSearchBarLocalizationString(this.activeFilter);
     this.go();
     this.go();
```

Comment:
Looks like `this.go()`  is duplicated here (I think I did this in the initial draft🙈 )

Reply:
[[bug] Remove duplicate this.go() call](https://github.com/bitwarden/web/pull/1595/commits/3a29ead2e5f33b0b7413627e6afcf2f996427f40)